src/Uno.UI/Controls/BindableImageView.Android.cs

```diff
@@ -270,6 +270,7 @@ private void OnUriSourceChanged()
 				var sampleSize = CalculateInSampleSize(options, _targetWidth, _targetHeight);
 
 				options.InJustDecodeBounds = false;
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Restore the warning after `InPurgeable`

Reply:
Wasn't supposed to be in the commit.  Will be removed.